tests/fixtures/mock_senders.js

```diff
@@ -21,4 +21,7 @@ module.exports = {
 			});
 		}
 	},
+	MockSenderPromise: function () {
```

Comment:
This method appears to be unused. Does it need to stay here? What was your intention with it?